org.opent2t.sample.hub.superpopular/com.nest.hub/js/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "opent2t-translator-com-nest-hub",
-  "version": "1.0.2",
+  "version": "0.2.0",
```

Comment:
>version": "0.2.0", [](start = 3, length = 18)

I don't think we can go backwards in version numbers, so some of the translators are stuck at >1.0.0 unfortunately.  Otherwise, "npm install opent2t-translator-com-nest-hub" will install an old version (the higher number).
For consistency, the minor (second) build number should be 2 for the new model, so this translator should be 1.2.0